format.md

```diff
@@ -49,51 +49,72 @@ The terminology for array types in this section is based on the
 GeoArrow proposes a packed columnar data format for the fundamental geometry
 types, using packed coordinate and offset arrays to define geometry objects.
 
-The inner level is always a FixedSizeList array storing the interleaved
-coordinate values (`[x, y, x, y, ...]`, physically stored as a single array
-of length `n_dim * n_coords`). For any geometry type except Point, this inner
-level is nested in one or multiple
+The inner level is always an array of coordinates. For any geometry type except
+Point, this inner level is nested in one or multiple
 [variable sized list](https://arrow.apache.org/docs/format/Columnar.html#variable-size-list-layout)
 arrays. In practice, this means we have additional arrays storing the offsets
 that denote where a new geometry, a new geometry part, or a new polygon ring
 starts.
 
-**Point**: `FixedSizeList<double>[n_dim]`
+This specification supports coordinates encoded as a Struct array storing the
+coordinate values as separate arrays (i.e., `x: [x, x, ...], y: [y, y, y, ...]`)
+and a FixedSizeList of interleaved values (i.e., `[x, y, x, y, ...]`). As
+implementations evolve, this specification may grow to support other coordinate
+representations or shrink to support only one if supporting multiple
+representations becomes a barrier to adoption.
 
-For an array of Point geometries, only a single array is used of interleaved
-coordinates. `n_dim` can be 2, 3, or 4 depending on the dimensionality of the
-geometries, and the field name of the list should be "xy", "xyz" or "xyzm",
-reflecting the dimensionality.
+**Coordinate (Struct)**: `Struct<x: double, y: double, [z: double, [m: double>]]`
+
+An array of coordinates can be stored as a Struct array containing two or more
+child double arrays with names corresponding to the dimension represented by
+the child. The first and second child arrays must represent the x and y
+dimension; where z and m dimensions are both included, the z dimension must
+preceed the m dimension.
+
+**Coordinate (FixedSizeList)**: `FixedSizeList<double>[n_dim]`
```

Comment:
```suggestion
**Coordinate (interleaved)**: `FixedSizeList<double>[n_dim]`
```

? (depending on what we want this term between brackets to mean, but in general I think it will be easier to talk about "interleaved coordinates" compared to "fixed-size list coordinates")